packages/patrol/darwin/Classes/AutomatorServer/Automator/IOSAutomator.swift

```diff
@@ -346,6 +346,19 @@
       }
     }
 
+    // MARK: Volume settings
+    func pressVolumeUp() throws {
+      runAction("pressing volume up") {
+        self.device.press(XCUIDevice.Button.pressVolumeUp)
+      }
```

Comment:
Acording to the [documentation](https://developer.apple.com/documentation/xctest/xcuidevice/button) this is not available on simulator. I think we should check that before calling just as we do for `runControlCenterAction`